rackcity/views/user_views.py

```diff
@@ -213,6 +213,131 @@ def who_am_i(request):
     return JsonResponse(serializer.data, status=HTTPStatus.OK)
 
 
+@api_view(['GET'])
+def i_am_admin(request):
+    """
+    Returns whether logged in user is an admin user.
+    """
+    if(request.user.is_staff):
+        return JsonResponse({"is_admin": True})
+    else:
+        return JsonResponse({"is_admin": False})
```

Comment:
moved this johnson from models to users where it belongs

Reply:
Did you just.......... `if true return true`

Reply:
Also I imagine it defaults to `status=HTTPStatus.OK` but idk if we want to say that explicitly?

Reply:
Did you just.......... `if true return true`

Reply:
Also I imagine it defaults to `status=HTTPStatus.OK` but idk if we want to say that explicitly?

Reply:
reverse uno card